apis/core/v1beta1/openstackcontrolplane_webhook.go

```diff
@@ -155,6 +155,10 @@ func (r *OpenStackControlPlane) checkDepsEnabled(name string) string {
 		if !((r.Spec.Mariadb.Enabled || r.Spec.Galera.Enabled) && r.Spec.Memcached.Enabled && r.Spec.Keystone.Enabled) {
 			reqs = "MariaDB or Galera, Memcached, Keystone"
 		}
+	case "Barbican":
+		if !((r.Spec.Mariadb.Enabled || r.Spec.Galera.Enabled) && r.Spec.Memcached.Enabled && r.Spec.Keystone.Enabled) {
```

Comment:
IIRC, Barbican does not depend or use Memcached in any way.

Reply:
fixed